packages/vercel-edge/src/index.ts

```diff
@@ -75,6 +75,11 @@ export {
   startSpanManual,
   continueTrace,
   metrics,
+  functionToStringIntegration,
+  inboundFiltersIntegration,
+  linkedErrorsIntegration,
+  moduleMetadataIntegration,
+  requestDataIntegration,
```

Comment:
Do we want to export these here? Previously, these were not accessible directly from `@sentry/vercel-edge.